server/svix-server/src/core/cryptography.rs

```diff
@@ -74,7 +74,7 @@ impl Encryption {
             let nonce = XNonce::from_slice(&nonce);
             let mut ciphertext = cipher
                 .encrypt(nonce, data)
-                .map_err(|_| crate::error::Error::Generic("Encryption failed".to_string()))?;
+                .map_err(|_| crate::err_generic!("Encryption failed"))?;
```

Comment:
Is the `crate::` prefix necessary here (or line 93)?

Reply:
Nope (same reason as before). Removed